firebase-firestore/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Unreleased
 
 # 21.4.2
-- [fixed] Remove Guava dependency from the SDK. This should eliminate crashes caused
-  by user App depending on a wrong flavor of Guava (#1125).
+- [changed] Removed Guava dependency from the SDK. This change is the first
```

Comment:
The "first step" in here makes it seem like there will be an active project to remove Guava from gRPC, but that's essentially a nonstarter, isn't it?